docs/README.md

```diff
@@ -1,10 +1,59 @@
 [Return to Project Root](../)
 
 # User Documentation
-tbd
+sail-operator manages the lifecycle of your Istio control planes and aims to become a vehicle for automated integration of Istio with compatible third-party software. Instead of creating a new configuration schema, sail-operator APIs are built around Istio's helm chart APIs. All installation and configuration options that are exposed by Istio's helm charts are available through the sail-operator CRDs' `values` fields.
```

Comment:
In one of the other PRs, it was agreed that we should use the convention "Sail Operator" (or sail operator) instead of "sail-operator". Accordingly the Readme was updated. Hope we can use the same convention in this doc as well.

Reply:
Oh, I didn't know - making the change